shared/js/textChatStore.js

```diff
@@ -187,6 +189,52 @@ loop.store.TextChatStore = (function() {
           }
         });
       }
+    },
+
+    /**
+     * Handles receiving information about the room context due to a change of the tabs
+     *
+     * @param  {sharedActions.updateRoomContext} actionData
+     */
+    updateRoomContext: function(actionData) {
+      // Firstly, check if there is a previous context tile, if not, create it
+      var contextTile = null;
+      var i = this._storeState.messageList.length - 1;
+      for (i; i >= 0; i--) {
```

Comment:
I think I'd rather keep these together on one line. The line length seems reasonable, especially compared to the line below.